tmt.spec

```diff
@@ -140,6 +143,13 @@ package to have all available plugins ready for testing.
 export SETUPTOOLS_SCM_PRETEND_VERSION=%{version}
 %pyproject_wheel
 
+# Build the man files
```

Comment:
Would `hatch run docs:man` work here?

Reply:
Nope, `hatch` needs internet connection and it does venv manipulation which we should not be doing in a spec file. I think it is best to duplicate these for a bit until we design these a bit better (we don't need to do all of these manipulations when we have `include` directive)